Makefile

```diff
@@ -29,15 +34,21 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	$(PIP_COMPILE) -o requirements/pip-tools.txt requirements/pip-tools.in
 	$(PIP_COMPILE) -o requirements/base.txt requirements/base.in
 	$(PIP_COMPILE) -o requirements/test.txt requirements/test.in
+	$(PIP_COMPILE) -o requirements/tox.txt requirements/tox.in
+	# Let tox control the Django version for tests
+	grep -e "^django==" requirements/test.txt > requirements/django.txt
```

Comment:
imo this is not necessary since we have to define manually the version

Reply:
Done